primitives/linux-port/primitiveprocessor.h

```diff
@@ -482,16 +484,18 @@ T getNullValue(uint8_t type)
     case execplan::CalpontSystemCatalog::DOUBLE:
     case execplan::CalpontSystemCatalog::UDOUBLE: return joblist::DOUBLENULL;
 
-    case execplan::CalpontSystemCatalog::CHAR:
-    case execplan::CalpontSystemCatalog::VARCHAR:
     case execplan::CalpontSystemCatalog::DATE:
     case execplan::CalpontSystemCatalog::DATETIME:
     case execplan::CalpontSystemCatalog::TIMESTAMP:
     case execplan::CalpontSystemCatalog::TIME:
     case execplan::CalpontSystemCatalog::VARBINARY:
     case execplan::CalpontSystemCatalog::BLOB:
+    case execplan::CalpontSystemCatalog::CHAR:
```

Comment:
@mariadb-SergeyZefirov according to TypeHandlerChar::getNullValueForType CHARs are stored as dicts when their width is > 8, so we would have to handle them in getNullValue<128>(uint8_t type). But @drrtuy says that this case is impossible.